core/src/main/java/org/zenoss/zep/impl/TriggerPlugin.java

```diff
@@ -848,7 +848,7 @@ protected synchronized void processSpool(long processCutoffTime) {
 
                 // These should have been deleted when the event was run through the TriggerPlugin when the status
                 // changed, but just in case delete them as the event is now in a closed state.
-                if (!OPEN_STATUSES.contains(status)) {
+                if (!OPEN_STATUSES.contains(status) || status == EventStatus.STATUS_ACKNOWLEDGED) {
```

Comment:
Jason is right. This would disable notifications for ack'ed events. Instead check if the event is ack'ed when we schedule the repetition in line 884. If it is ack you don't schedule the repetition.